go/roothash/api/commitment/pool.go

```diff
@@ -449,16 +449,19 @@ func (p *Pool) ProcessCommitments(didTimeout bool) (OpenCommitment, error) {
 			return nil, ErrMajorityFailure
 		}
 
+		// If we already have the proposer commitment, we can try to resolve early based on majority
+		// vote. Otherwise we need to wait for the proposer commitment.
+		if proposerCommit == nil {
```

Comment:
In resolution mode, if we don't have the proposer commitment and timeout happened, we return `ErrInsufficientVotes` instead of `ErrNoProposerCommitment`. 
Could be fixed by moving the first block in `if didTimeout {` to the end.